webpack/InsightsCloudSync/Components/InsightsSettings/InsightsSettings.js

```diff
@@ -12,9 +12,17 @@ class InsightsSettings extends Component {
 
   render() {
     const { insightsSyncEnabled, setInsightsSyncEnabled } = this.props;
+    const insightsStyles = {
+      border: '1px solid #ededed',
+      borderRadius: '4px',
+      padding: '10px',
+      marginBottom: '15px',
+      float: 'right',
+    };
+
```

Comment:
currently when using foreman router there's no css loaded, from the next version importing scss will be supported.

this is a workaround to get our styles to work.